recipes/python-casacore/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "python-casacore" %}
+{% set version = "2.1.2" %}
+{% set sha256 = "b4ea03d1b3d72ddfa67c2b4b572dd92f39f806b090c49b4715f982ac9ecba2b1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/casacore/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - boost-python.patch
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - gcc  # [osx]
+    - boost 1.63.*  # note this is intentionally NOT the latest version; synced with casacore which has special needs
```

Comment:
I the long run we need a way to tell the pinning script to ignore feedstocks we don't want to update the pin automatically.
(Not a problem now b/c the pinning script is broken :smile:)